### DISCUSSION TITLE
npm info returns 0-exit code on package versions that don't exist

### DISCUSSION THREAD

**Discussion by ThisIsMissEm** [npm/feedback#697] [CLI]
Currently when you do `npm info` on a specific package version, if it doesn't exist, then the command exits with a no output and a zero status code, when it should probably exit with a 1 status code.

**Comment by ljharb** [comment]
This seems like a simple bug in the CLI - you may want to file it there.

**Reply by ThisIsMissEm** [reply]
Okay, have opened it: https://github.com/npm/cli/issues/4964
